[MTM-58756] Release notes: removal deprecated JWT with Cumulocity pro… (#2520)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-596-removal-of-deprecated-jwt-login-mode.md

---
date: 2024-10-15
title: Removal of deprecated JWT login mode
product_area: Platform services
change_type:
  - value: change-inv-3bw8e

>     label: Announcement

Reviewer: This is not an announcement, but an improvement I would say.

Author: Within a team, we talked that, it should go as an announcement. We announce that deprecated JWT can not be used anymore.

component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-58756
version: 10.20.596.0
---

> Deprecated JWT authentication with {{< product-c8y-iot >}} has been removed.

Reviewer: ```suggestion
The deprecated JWT authentication with {{< product-c8y-iot >}} has been removed.
```

> The deprecated JWT authentication with {{< product-c8y-iot >}} has been removed.

Reviewer 2: Minor: The deprecated JWT authentication method has been removed from {{< product-c8y-iot >}}. - that seems to be clearer for me but I see that @BeateRixen approved that so I put it as optional :)

> This change will only impact you if your organization used an early version of SSO on {{< product-c8y-iot >}}.

Reviewer: ```suggestion
This change will only impact you if your organization used an early version of single sign on (SSO) on {{< product-c8y-iot >}}.
```

> More details can be seen in the early published [announcement](https://cumulocity.com/docs/change-logs/?#cumulocity-10-20-removal-of-deprecated-jwt-login-mode)

Reviewer: ```suggestion
For details, refer to the earlier [announcement](https://cumulocity.com/docs/change-logs/?#cumulocity-10-20-removal-of-deprecated-jwt-login-mode).
```
